src/main/java/week2/NewYearChaos.java

```diff
@@ -0,0 +1,61 @@
+package week2;
+
+import java.util.Scanner;
+
+public class NewYearChaos {
+
+    // Complete the minimumBribes function below.
+    static void minimumBribes(int[] q) {
+        int temp = 0, count = 0, check =0;
+        int[] bribe = new int[1000000];
+        boolean isSorted = false;
+
+        while(isSorted != true){
+            isSorted = true;
+            for(int i=q.length-1; i>0 ; --i){
+                if(q[i] < q[i-1]){
+                    temp = q[i-1];
+                    q[i-1] = q[i];
+                    q[i] = temp; 
+                    bribe[q[i]]++;
+                    count++;
+                    if(bribe[q[i]] > 2) {
+                        System.out.println("Too chaotic");
+                        check++;
+                        break;
+                    }
+                    isSorted = false;
```

Comment:
Code modularity and code link at top is still missing on every problem solution.

Reply:
Sure will keep in mind